### DISCUSSION TITLE
Portfolios For Inspiration

### DISCUSSION THREAD

**Discussion by anushbhatia** [smaranjitghose/awesome-portfolio-websites#1190] [Ideas]
Drop out some of the inspiration for portfolios that you like

Also can mention:
1. Sections you love in that
2. Any specific feature should be included in out project

**Comment by nivinvysakh** [comment]
Hai @anushbhatia  I have build a Retro TUI & CLI Developer Portfolio Template powered by Astro 5 & Tailwind CSS v4. I was thinking About adding my website into your list as a Pull Request  . 

Features :
 - Its a clean minimal Terminal Style Portfolio Website built using Astro.js and Tailwind.css 
 - It currently has 5 Themes that you can choose which theme do you love 
 - It as a Easter Egg Matrix Style Effect
 - Any one can change or edit the content in one single file `src/data/portfolio.ts` without touching the structure or core files . 

I have attached Some Screenshots of some themes which I do like  

> Amber CRT
<img width="1701" height="936" alt="Amber_CRT" src="https://github.com/user-attachments/assets/d321599a-9eb2-4d5b-bf68-72afd2e3e7e9" />

> Dracula_Synth the one I loved Most 
<img width="1692" height="944" alt="Dracula_Synth" src="https://github.com/user-attachments/assets/f19dd9e4-9cce-44bc-8bd7-5762f31c2fed" />


Repo Link : https://github.com/nivinvysakh/astro-tui-portfolio
Live Demo Link : https://astro-tui-portfolio.netlify.app

cheers ,
Nivin xoxo :)
